debian/rules

```diff
@@ -16,27 +14,47 @@ endif
 # make .PHONY all the targets that have name collisions with the scripts
 # see http://www.debian.org/doc/manuals/maint-guide/dreq.en.html#rules
 .PHONY: clean build install
-# Apparently, the above isn't enough because of the "%" target. Make the problematic targets explicit
-clean:
-	exec dh $@
-build:
-	exec dh $@
-install:
+# The above isn't enough because of the "%" target. Make the problematic targets explicit:
+clean build install:
 	exec dh $@
 # Handle all other targets in the usual way.
-# The --parallel flag to dh doesn't seem to have the intended effect
-# so leave it out.
 %:
 	exec dh $@
 
-libass_config:
+projects := $(shell ./get-config -- config_projects)
+
+mujs_config:
+	./build --mode=config mujs
+
+mujs_build: mujs_config
+	./build --mode=build mujs -- $(MFLAGS)
+
+shaderc_config:
+	./build --mode=config shaderc
+
+shaderc_build: shaderc_config
+	./build --mode=build shaderc
+
+crossc_config:
+	./build --mode=config crossc
+
+crossc_build: crossc_config
+	./build --mode=build crossc -- $(MFLAGS)
+
+fribidi_config:
+	./build --mode=config fribidi
+
+fribidi_build: fribidi_config
+	./build --mode=build fribidi -- $(MFLAGS)
+
+libass_config: $(patsubst %,%_build,$(filter fribidi,$(projects)))
 	./build --mode=config libass
 
 libass_build: libass_config
 	./build --mode=build libass -- $(MFLAGS)
 
```

Comment:
Should I wait with the merge till you add support for libaom at the debian thingy too?